ibis/backends/base/sql/alchemy/datatypes.py

```diff
@@ -82,6 +82,8 @@ def table_from_schema(name, meta, schema, database: str | None = None):
 # TODO(cleanup)
 ibis_type_to_sqla = {
     dt.Null: sa.types.NullType,
+    dt.Enum: sa.Enum,
+    dt.Category: sa.Enum,
```

Comment:
Eventually I want to deprecate `dt.Category` in favor of `dt.Enum` only, so ideally we don't add net new functionality that uses `Category`.